.github/workflows/tests.yml

```diff
@@ -334,42 +334,6 @@ jobs:
           echo "Deleting $SOFTWARE_NAME"
           coiled env delete $SOFTWARE_NAME
 
-  report:
-    name: report
-    needs: [runtime, benchmarks, stability]
-    if: |
-      always()
-      && github.event_name != 'pull_request'
-      && github.repository == 'coiled/coiled-runtime'
-      && (needs.runtime.result == 'failure' ||
-          needs.benchmarks.result == 'failure' ||
-          needs.stability.result == 'failure')
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash
-    steps:
-      - uses: actions/checkout@v2
-      - name: Report failures
-        uses: actions/github-script@v3
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            const title = "⚠️ CI failed ⚠️"
-            const workflow_url = `https://github.com/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID}`
-            const issue_body = `[Workflow Run URL](${workflow_url})`
-            const variables = {
-                owner: context.repo.owner,
-                name: context.repo.repo,
-                creator: "github-actions[bot]"
-            }
-            github.issues.create({
-                owner: variables.owner,
-                repo: variables.name,
-                body: issue_body,
-                title: title,
-            })
-
   process-results:
     needs: [runtime, benchmarks, stability]
     name: Combine separate benchmark results
```

Comment:
So, if we wanted to run the regression check on PRs, I think we'd have to remove the `github.ref` check on `process-results`, but then add it to the "Upload benchmark db" step. Then `regressions` could just `always()` run.